src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h

```diff
@@ -1200,6 +1206,12 @@ struct YarpSensorBridge::Impl
                 std::cerr << logPrefix << " NAN values read from encoders interface, use previous measurement" << std::endl;
                 return false;
             }
+
+            if (std::binary_search(controlBoardRemapperMeasures.jointCurrents.begin(), controlBoardRemapperMeasures.jointCurrents.end(), NAN))
+            {
+                std::cerr << logPrefix << " NAN values read from current control interface, use previous measurement" << std::endl;
+                return false;
+            }
```

Comment:
I think that this should be changed according to #191. Indeed, by reading this code I opened #188 :grin: